app/helpers/components_helper.rb

```diff
@@ -12,6 +12,10 @@ def maintaining_iterations_summary_table(iterations)
                     })
   end
 
+  def notification_icon(user)
+    react_component("notification-icon", { count: user.notifications.count })
```

Comment:
I think this should be `user.notifications.unread.count` and the tests etc updated accordingly.

Reply:
Thanks. Done.